### ISSUE TITLE
Errno::ECONNRESET in LiteCable::Server::ClientSocket::Base#close_socket

### ISSUE THREAD

**Issue by cyberarm** [palkan/litecable#73]
### Tell us about your environment

**Ruby version:** ruby 3.2.0 (2022-12-25 revision a528908271) [x64-mingw-ucrt]

**`litecable` gem version:** 0.8.0

**`anycable` gem version:**

**`grpc` gem version:**

### What did you do?
Let computer sleep overnight and woke it up.

### What did you expect to happen?
Keep on running.

### What actually happened?
Fatal socket error:
```ruby
Errno::ECONNRESET
```
```txt
C:/Users/cyber/Rubies/Ruby32-x64/lib/ruby/gems/3.2.0/gems/litecable-0.8.0/lib/lite_cable/server/client_socket/base.rb:115:in `write': An existing connection was forcibly closed by the remote host. (Errno::ECONNRESET)
        from C:/Users/cyber/Rubies/Ruby32-x64/lib/ruby/gems/3.2.0/gems/litecable-0.8.0/lib/lite_cable/server/client_socket/base.rb:115:in `close_socket'
        from C:/Users/cyber/Rubies/Ruby32-x64/lib/ruby/gems/3.2.0/gems/litecable-0.8.0/lib/lite_cable/server/client_socket/base.rb:107:in `close!'
        from C:/Users/cyber/Rubies/Ruby32-x64/lib/ruby/gems/3.2.0/gems/litecable-0.8.0/lib/lite_cable/server/client_socket/base.rb:92:in `close'
        from C:/Users/cyber/Rubies/Ruby32-x64/lib/ruby/gems/3.2.0/gems/litecable-0.8.0/lib/lite_cable/server/client_socket/base.rb:83:in `block in listen'
```

https://github.com/palkan/litecable/blob/43244077fa964e39cef1b6493c5c71a6a4b9c3b0/lib/lite_cable/server/client_socket/base.rb#L112-L120

**Comment by palkan** [comment]
Thanks for the report. Fixed and released on 0.8.1.

**Comment by cyberarm** [comment]
Thanks for the quick fix! 🙂
